### ISSUE TITLE
There is not version2.5 selection on Ansible

### ISSUE THREAD

**Issue by qiaofeng1227** [Websoft9/ansible-haproxy#1]
**Describe the bug**

## Deployment method

- [ ] subscription image from MarketPlace of Cloud
- [ ] All-in-one Installer
- [ ] ansible-playbook cli
- [x] mcloud
- [ ] create instance from private image

## Bug source

- [x] Bugs from installation 
- [ ] Bugs from first use
- [ ] Other (You should describe it)   

## System Environment

**Application Version**

e.g v4.5

**OS**

- [ ] Ubuntu18.x
- [x] Ubuntu20.x
- [ ] CentOS7.x
- [ ] CentOS8.x
- [ ] AmazonLinux2
- [ ] OracleLinux7.8
- [ ] OracleLinux8.2
- [ ] Other (You should describe it) 

**Cloud PlatForm**

- [ ] Azure
- [ ] Aws
- [x] 阿里云
- [ ] 腾讯云
- [ ] 华为云
- [ ] AlibabaCloud
- [ ] HUWEICloud
- [ ] Other (You should describe it) 

### Bug Context

Bug context includes bug error, error logs and service status, it is very important for resolving issue

```
# bug error
(paste here) 

# service status 
(paste here) 

# error logs
(paste here) 
```

### Bug Screenshot

paste screenshot here

## Your suggestion

**The reason of Bug**

You can describe the reason if you have found it

**Your solution for Bug**

You can describe your solution here

**Additional context**

You can add any other context about the problem here.

**Comment by qiaofeng1227** [comment]
add haproxy2.5 install selection on Ansible.

**Comment by qiaofeng1227** [comment]
```
tccli cvm RunInstances             --InstanceChargeType SPOTPAID             --InstanceMarketOptions '{"MarketType":"spot","SpotOptions":{"MaxPrice":"5"}}'             --Placement '{"Zone":"ap-hongkong-2"}'             --InstanceType S2.MEDIUM8 --ImageId img-l8og963d             --SystemDisk '{"DiskType":"CLOUD_PREMIUM", "DiskSize":50}'             --InstanceCount 1 --InstanceName websoft9-linuxe-CentOS7.9             --LoginSettings '{"KeyIds":["skey-2x4l51cn"]}'             --SecurityGroupIds '["sg-hx3q66r9"]'             --InternetAccessible '{"InternetMaxBandwidthOut":100}' --filter InstanceIdSet[0] 
^CTraceback (most recent call last):
  File "build.py", line 46, in <module>
    w9meta_data.action_create_vm() 
  File "/data/mcloud/W9Meta_data.py", line 532, in action_create_vm
    Cloud_cli.execute_CommandReturn(self.cloudmeta[self.cloud_select]['vm_createVM_template'])
  File "/data/mcloud/W9System.py", line 229, in execute_CommandReturn
    out_str=subprocess.getstatusoutput(cmd_str)
  File "/usr/lib64/python3.6/subprocess.py", line 537, in getstatusoutput
    data = check_output(cmd, shell=True, universal_newlines=True, stderr=STDOUT)
  File "/usr/lib64/python3.6/subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "/usr/lib64/python3.6/subprocess.py", line 425, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib64/python3.6/subprocess.py", line 850, in communicate
    stdout = self.stdout.read()
  File "/usr/lib64/python3.6/codecs.py", line 318, in decode
    def decode(self, input, final=False):

```

**Comment by QiuJiaHon** [comment]
**fail to install Haproxy 2.5 on Ubuntu on Aliyun** 
```
fatal: [47.242.238.65]: FAILED! => {"changed": true, "cmd": "apt-get install -y --no-install-recommends software-properties-common\nadd-apt-repository -y ppa:vbernat/haproxy-2.5\n", "delta": "0:06:36.879753", "end": "2021-12-06 13:18:01.563529", "msg": "non-zero return code", "rc": 1, "start": "2021-12-06 13:11:24.683776", "stderr": "", "stderr_lines": [], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nsoftware-properties-common is already the newest version (0.99.9.8).\n0 upgraded, 0 newly installed, 0 to remove and 120 not upgraded.\nError: retrieving gpg key timed out.", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information...", "software-properties-common is already the newest version (0.99.9.8).", "0 upgraded, 0 newly installed, 0 to remove and 120 not upgraded.", "Error: retrieving gpg key timed out."]}
```

**Comment by qiaofeng1227** [comment]
```
TASK [role_cloud : Add to fstab] *********************************************************************************************************************************************************************************************************************************************
changed: [8.217.36.196]
included: /tmp/20211207105151-ansible-haproxy/ansible-haproxy/roles/role_haproxy/tasks/Debian.yml for 8.217.36.196

TASK [role_haproxy : Prepare for install] ************************************************************************************************************************************************************************************************************************************
fatal: [8.217.36.196]: FAILED! => {"changed": true, "cmd": "apt-get install -y --no-install-recommends software-properties-common\nadd-apt-repository -y ppa:vbernat/haproxy-2.5\n", "delta": "0:08:47.303557", "end": "2021-12-07 11:33:16.549012", "msg": "non-zero return code", "rc": 1, "start": "2021-12-07 11:24:29.245455", "stderr": "", "stderr_lines": [], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nsoftware-properties-common is already the newest version (0.99.9.8).\n0 upgraded, 0 newly installed, 0 to remove and 120 not upgraded.\nCannot add PPA: 'ppa:~vbernat/ubuntu/haproxy-2.5'.\nERROR: '~vbernat' user or team does not exist.", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information...", "software-properties-common is already the newest version (0.99.9.8).", "0 upgraded, 0 newly installed, 0 to remove and 120 not upgraded.", "Cannot add PPA: 'ppa:~vbernat/ubuntu/haproxy-2.5'.", "ERROR: '~vbernat' user or team does not exist."]}

PLAY RECAP *******************************************************************************************************************************************************************************************************************************************************************

```

**Comment by qiaofeng1227** [comment]
```
root@iZj6ceaosp5c45ljfxmv2fZ:/etc# add-apt-repository -y ppa:vbernat/haproxy-2.5
Error: retrieving gpg key timed out.

```
